ext/lvgl/lv_conf.h.in

```diff
@@ -0,0 +1,100 @@
+/**
+ * @file lv_conf.h
+ * Configuration file for v7.11.0
+ */
+
+#ifndef LV_CONF_H
+#define LV_CONF_H
+/* clang-format off */
+
+#include <stdint.h>
+
+/* A header file that overwrites with local project settings. */
+#if __has_include(<lv_conf_local.h>)
+#  include <lv_conf_local.h>
+#endif
+
+/*====================
+   Graphical settings
+ *====================*/
+
+/* Maximal horizontal and vertical resolution to support by the library.*/
+#define LV_HOR_RES_MAX          ({{ options.resolution_horizontal_max }})
+#define LV_VER_RES_MAX          ({{ options.resolution_vertical_max }})
+
+/* Color depth:
+ * - 1:  1 byte per pixel
+ * - 8:  RGB332
+ * - 16: RGB565
+ * - 32: ARGB8888
+ */
+#define LV_COLOR_DEPTH     {{ options.color_depth }}
+
+/* Enable anti-aliasing (lines, and radiuses will be smoothed) */
+#ifndef LV_ANTIALIAS
+#define LV_ANTIALIAS        0
+#endif
+
+/* Dot Per Inch: used to initialize default sizes.
+ * E.g. a button with width = LV_DPI / 2 -> half inch wide
+ * (Not so important, you can adjust it to modify default sizes and spaces)*/
+#define LV_DPI              {{ options.dpi }}     /*[px]*/
+
+/* Type of coordinates. Should be `int16_t` (or `int32_t` for extreme cases) */
+typedef int16_t lv_coord_t;
+
+/*=========================
+   Memory manager settings
+ *=========================*/
+
+/* 1: use custom malloc/free, 0: use the built-in `lv_mem_alloc` and `lv_mem_free` */
+#define LV_MEM_CUSTOM      1
+
+/*==================
+ * Feature usage
+ *==================*/
+
+/* 1: Enable GPU interface*/
+#ifndef LV_USE_GPU
+#define LV_USE_GPU              0   /*Only enables `gpu_fill_cb` and `gpu_blend_cb` in the disp. drv- */
+#endif
+
+/* 1: Enable file system (might be required for images */
+#ifndef LV_USE_FILESYSTEM
+#define LV_USE_FILESYSTEM       0
+#endif
+
```

Comment:
We could also remove all this lbuild options and instead let the user configure them through `<lv_conf_local.h>`. What do you think?

Reply:
Yes, I would prefer that since the lbuild options don't add any additional benefits (ie. like other modules being able to query display size). I'll add a few more fixups.

Reply:
I've moved all config options out of lbuild into `lv_conf_local.h` incl. logging, since it's weird to have to regenerate modm again just for that one option. I like this solution very much, it's very clean and minimal.